cvrf2csaf/cvrf2csaf.py

```diff
@@ -34,6 +34,9 @@ class DocumentHandler:
     5. Combining it to the final JSON and writing the result to a file
     """
 
+    TOLERATED_ERRORS_SUBSTR = ["}ScoreSetV3': This element is not expected. Expected is one of ( {http://docs.oasis-open.org/csaf/ns/csaf-cvrf/v1.2/vuln}ScoreSetV2, {http://docs.oasis-open.org/csaf/ns/csaf-cvrf/v1.2/vuln}ScoreSetV3 ).",
```

Comment:
Nit: `"}ScoreSetV3':`  &rightarrow; `"ScoreSetV3:`?

Reply:
I guess it is to see that it is a namespace error...

Reply:
@sthagen The original error message according to https://github.com/csaf-tools/CVRF-CSAF-Converter/pull/56#discussion_r808070742 was:

`Element '{https://www.first.org/cvss/cvss-v3.1.xsd}ScoreSetV3': This element is not expected. Expected is one of ( {http://docs.oasis-open.org/csaf/ns/csaf-cvrf/v1.2/vuln}ScoreSetV2, {http://docs.oasis-open.org/csaf/ns/csaf-cvrf/v1.2/vuln}ScoreSetV3 )`

So this looks okay to me.

Reply:
@tschmidtb51 exactly, I just picked the longest fixed substring containing this particular error. I will unify both substrings (add '}' to the second one).

Reply:
🙈 I was assuming (wrongly) that this was an output string and not a substring. Sorry for the noise.